app/scripts/metamask-controller.js

```diff
@@ -450,6 +451,7 @@ export default class MetamaskController extends EventEmitter {
     this.appStateController = new AppStateController({
       addUnlockListener: this.on.bind(this, 'unlock'),
       isUnlocked: this.isUnlocked.bind(this),
+      isAttemptingLogin: isManifestV3,
```

Comment:
Why is initial state set like this? 
if isManifestV3 is true, then the process triggered is setting this state to true anyway. 
That is, unless password was used, in which case nothing happens to the state and it stays set to true forever.

Reply:
I set it to `isManifestV3` because we only care about this property, at the moment, if we're using MV3.  If we're using MV3, we know that we're going to perform the async action of checking storage.